src/libecalc/presentation/yaml/domain/time_series_resource.py

```diff
@@ -0,0 +1,143 @@
+import re
+from datetime import datetime
+from math import isnan
+from typing import Iterable, List, Union
+
+from pandas.errors import ParserError
+from typing_extensions import Self
+
+from libecalc.common.errors.exceptions import InvalidColumn, InvalidHeader, InvalidResource, NoColumns
+from libecalc.common.string.string_utils import get_duplicates
+from libecalc.presentation.yaml.mappers.variables_mapper.time_series_collection_mapper import parse_time_vector
+from libecalc.presentation.yaml.resource import Resource
+from libecalc.presentation.yaml.yaml_keywords import EcalcYamlKeywords
+
+
+class InvalidTimeSeriesResource(InvalidResource):
```

Comment:
should we use the convention for expceptions to end in Exception or Error? I think we should